test/bucket.ts

```diff
@@ -3237,4 +3238,42 @@ describe('Bucket', () => {
       );
     });
   });
+  describe('disableAutoRetryConditionallyIdempotent_', () => {
```

Comment:
is there a test for making sure autoRetry stays true? we should cover at least one case of that. otherwise, looks good. thanks, denis!

Reply:
Added!